sdks/cpp/common/include/Authorization.h

```diff
@@ -50,64 +51,57 @@ namespace common {
 /**
  * @brief Class for handling authorization information
  */
-class AuthzInfo {
-    using ParamDescriptor = catena::common::ParamDescriptor;
+class Authorizer {
   public:
+    /**
+     * @brief Special Authorizer object that disables authorization
+     */
+    static Authorizer kAuthzDisabled;
 
     /**
-     * @brief Construct a new AuthzInfo object
+     * @brief Construct a new Authorizer object
      * @param pd the ParamDescriptor of the object
      * @param scope the scope of the object
      */
-    AuthzInfo(const ParamDescriptor& pd, const std::string& scope)
-      : pd_{pd}, clientScope_{scope}{}
+    Authorizer(const std::vector<std::string>& clientScopes)
+        : clientScopes_{clientScopes} {}
+
+    Authorizer(const Authorizer&) = delete;
```

Comment:
please add doxygen comments to say that Authorizer doesn't have copy semantics, and that Authorizer does have move semantics.